net-p2p/go-ipfs/go-ipfs-0.4.23.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit bash-completion-r1 go-module golang-build systemd
+
+DESCRIPTION="Main implementation of IPFS"
+HOMEPAGE="https://ipfs.io/"
+SRC_URI="https://dist.ipfs.io/go-ipfs/v${PV}/go-ipfs-source.tar.gz -> ${P}.tar.gz
+	https://raw.githubusercontent.com/ipfs/go-ipfs/v${PV}/misc/completion/ipfs-completion.bash -> ${P}.bash"
+EGO_PN="github.com/ipfs/go-ipfs"
+
+LICENSE="MIT"
```

Comment:
You'll need to add licenses of bundled deps as well. I think `go-license-detector` can identify them, but I haven't used it.